mitmproxy/addons/save.py

```diff
@@ -44,6 +46,17 @@ def start_stream_to_path(self, path, flt):
         self.stream = io.FilteredFlowWriter(f, flt)
         self.active_flows = set()
 
+    def add(self, flow):
+        try:
+            self.stream.add(flow)
+        except OSError as e:
+            if e.errno == errno.ENOSPC:
```

Comment:
Practically speaking it may be useful if we just exit on _every_ OSError? I mean in any case it indicates that we cannot save anymore, which is critical.

Reply:
Okay, but if exit on every OSError, then we might not understand why exactly the call failed. Is it better to add log the error string from OSError and then exit?

Reply:
Yes, that sounds great.

Reply:
Hello, I fixed the code as needed. Is this ok? If yes, Ill proceed with tests.